cds/modules/deposit/api.py

```diff
@@ -336,6 +384,18 @@ def delete(self, force=True, pid=None):
             video.delete(force=force)
         return super(Project, self).delete(force=force, pid=pid)
 
+    def get_report_number_sequence(self, **kwargs):
+        """Get the sequence generator for Projects."""
+        try:
+            year = self['date'].split('-')[0]
```

Comment:
This is probably better done using something like [arrow](http://crsmithdev.com/arrow) (which we use already), as our dates will be always in ISO format.

```python
arrow.get(self['date']).year
```

Probably we should also make `date` a field required ...